lib/opentelemetry-proto/build.rs

```diff
@@ -1,9 +1,100 @@
 use std::io::Error;
 
+// NOTE: Serde related attributes are copied from https://github.com/open-telemetry/opentelemetry-rust/
```

Comment:
Is this an issue wrt the CLA and licenses?

I initially started on an implementation and its ended up very similar (don't think tonic and serde offer an alternative way). Figured that copying it is easier as it makes it easier to sync if otel changes